fh-messaging/lib/mongo_functions/reduce_aggregated_apprequestsdest.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable*/
+//ts and emit are used for testing purposes
+var ts;
+var emit;
+function metrics_aggregated_apprequestdest_reduce(k, mbaasMetrics) {
+  // Take the domain and ts in the first vals entry as the reduced elements domain and ts
+  var result = {};
+
+
+  function forEach(iterable, cb) {
+    for (var i in iterable) {
+      if (iterable.hasOwnProperty(i)) {
+        cb(iterable[i], i);
+      }
+    }
+  }
+
+  forEach(mbaasMetrics, function(appMetrics) {
+    forEach(appMetrics, function(metrics, p) {
+      if (typeof result[p] !== 'undefined') {
+        result[p] += metrics;
+      } else {
+        result[p] = metrics;
+      }
+    });
+  });
+  return result;
+}
+
+exports.metrics_aggregated_apprequestdest_reduce = metrics_aggregated_apprequestdest_reduce;
+
+exports.setVars = function(tstamp,emitFunc) {
+  ts = tstamp;
+  emit = emitFunc;
+  console.log('timestamp=' + ts + ', emit=' + emit);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)